src/btree.c

```diff
@@ -2990,6 +2999,17 @@ btree_pack_can_fit_tuple(btree_pack_req *req, slice key, message data)
                 <= req->max_kv_bytes;
 }
 
+static void
+btree_pack_abort(btree_pack_req *req)
+{
+   btree_dec_ref_range(req->cc,
+                       req->cfg,
+                       req->root_addr,
+                       NULL_SLICE,
+                       NULL_SLICE,
+                       PAGE_TYPE_BRANCH);
+}
+
```

Comment:
`btree_pack_abort` is supposed to destroy the entire data structure. If there is more things to do, let me know.

Reply:
@rtjohnso -- Got it. Let me add the code.